### PR TITLE
[1LP][RFR][NOTEST] fix for historical data with unicode and str

### PR DESCRIPTION
**Pull request by digitronik**
Purpose or Intent
=================
- removed `repr` so compartable with `unicode` and `str`
- F401 'wrapanapi.exceptions.NotFoundError' imported but unused :+1: 

Shriver adding pytest:

{{ pytest: cfme/tests/candu/test_vm_graph.py cfme/tests/candu/test_host_graph.py }}

### CODE REVIEW

File: cfme/common/vm.py [hunk 1]
```diff
@@ -659,17 +658,19 @@ def wait_candu_data_available(self, timeout=600):
...
             interval: Data interval (hourly/ daily)
             back: back time interval from which you want data
         """
         ret = self.appliance.ssh_client.run_rails_command(
-            "\"vm = Vm.where(:ems_id => {}).where(:name => {})[0];\
-            vm.perf_capture({}, {}.ago.utc, Time.now.utc)\""
-            .format(self.provider.id, repr(self.name), repr(interval), back))
+            "\"vm = Vm.where(:ems_id => {prov_id}).where(:name => '{vm_name}')[0];\
+            vm.perf_capture('{interval}', {back}.ago.utc, Time.now.utc)\"".format(
+                prov_id=self.provider.id, vm_name=self.name, interval=interval, back=back
```
Comment (RonnyPfannschmidt) [inline cfme/common/vm.py]: **optional** for both of name and interval i would like to propose to use a `json.dump` to protect from bad characters

File: cfme/infrastructure/host.py [hunk 2]
```diff
@@ -418,17 +418,19 @@ def wait_candu_data_available(self, timeout=900):
...
             interval: Data interval (hourly/ daily)
             back: back time interval from which you want data
         """
         ret = self.appliance.ssh_client.run_rails_command(
-            "\"host = Host.where(:ems_id => {}).where(:name => {})[0];\
-            host.perf_capture({}, {}.ago.utc, Time.now.utc)\""
-            .format(self.provider.id, repr(self.name), repr(interval), back))
+            "\"host = Host.where(:ems_id => {prov_id}).where(:name => '{host_name}')[0];\
+            host.perf_capture('{interval}', {back}.ago.utc, Time.now.utc)\"".format(
+                prov_id=self.provider.id, host_name=self.name, interval=interval, back=back
```
Comment (RonnyPfannschmidt) [inline cfme/infrastructure/host.py]: duto

**Comment by mshriver** [comment]
why NOTEST? this method is used in tests, I added pytest line to include them.

**Comment by digitronik** [comment]
Now, it will take too much time to run :D

**Comment by mshriver** [comment]
PRT results in 59z showing that this method is executing cleanly and as expected.

**Pull request merged**